### PR TITLE
LCALS testing updates

### PR DESCRIPTION
**Pull request by daviditen**
Remove the --dataParMinGranularity=1000 option so it is a more direct
comparison against the OpenMP version.  This unfortunately causes the short
versions to take much more time.

Stop running the vectorizeOnly variant. It is effectively identical to the
serial variant, so is not interesting to run currently.  Remove it from the
perfkeys and graph files.

### CODE REVIEW

File: test/release/examples/benchmarks/lcals/LCALSMain.perfexecopts [hunk 1]
```diff
@@ -1 +1 @@
---dataParMinGranularity=1000
+--run_variantRawVectorizeOnly=false
```
Comment (ronawho) [inline test/release/examples/benchmarks/lcals/LCALSMain.perfexecopts]: Any reason to not just disable the vectorize only by default in the code?

Comment (daviditen) [inline test/release/examples/benchmarks/lcals/LCALSMain.perfexecopts]: Nothing too deep.  I can make that change.

**Comment by ronawho** [comment]
Looks good to m

**Comment by daviditen** [comment]
Thank you!

**Pull request merged**